vitest.base.unit.config.ts

```diff
@@ -20,7 +20,7 @@ export default defineConfig({
     ],
     reporters: process.env.GITHUB_ACTIONS
       ? ["verbose", "hanging-process", "github-actions"]
-      : ["verbose", "hanging-process"],
+      : [process.env.TEST_COMPACT_OUTPUT === "true" ? "basic" : "verbose", "hanging-process"],
```

Comment:
These variables works as flags so only their presence seems good enough to check. 

```suggestion
      : [process.env.TEST_COMPACT_OUTPUT ? "basic" : "verbose", "hanging-process"],
```